src/Components/NavBar/NavBar.js

```diff
@@ -1,10 +1,64 @@
 
+import { NavLink } from "react-router-dom"
+
 import "./_NavBar.scss"
+import rabbit from '../../images/rabbit.png'
 
 const NavBar = () => {
+//username, avatarUrl, setClicked, clicked for props
+  const handleChange = (event) => {
+    const value = event.target.value
+    // This function takes in the text input of the search bar and sets the input value to the local variable value. I am thinking that the search functionality/setting state will be handled from here.
+  }
+
+  // const isClicked = (clicked) => {
+  //     let homeButton
+  //     let watchlistButton
+  //   if(clicked === 'home') {
+  //       homeButton = 'hide'
+  //       watchlistButton = 'show-button'
+  //   } else if (clicked === 'watchlist') {
+  //     homeButton = 'show-button'
+  //     watchlistButton = 'hide'
+  //   } else if (clicked === 'details') {
+  //     homeButton = 'show-button'
+  //     watchlistButton = 'show-button'
+  //   }
+  // }
+
+  // Refactor the above function
 
   return (
-    <h1>nav bar</h1>
+    <div className="navbar-area">
+      <section className="user-info">
+        <img src={rabbit} alt='Picture of your avatar' />
+        <p>Courtney</p>
+      </section>
+      <section className="search-navigation">
+        <input 
+        type='text'
+        placeholder='Search'
+        name='search'
+        // value={this.state.value}
+        onChange={handleChange}
+        className='search-input'
+        />
+        <NavLink 
+        className="home-button" 
+        id='home-button' 
+        to='/' 
+        >
+          <p>Home</p>
+        </NavLink>
+        <NavLink 
+        className="watchlist-button" 
+        id='watchlist-button' 
+        to='/watchlist'
+        >
+          <p>My Watchlist</p>
+        </NavLink>
+      </section>
```

Comment:
Nice use of `<NavLink>`s! I don't have much experience with them, so I looked up the docs on them (see below). One thought is, have you thought about swapping the `<section>` element containing the `<NavLink>`s for a `<nav>` element and using an unordered list? Just thinking about accessibility. 

This is from the React docs on `<NavLink>`:
![Screen Shot 2023-02-07 at 6 13 35 PM](https://user-images.githubusercontent.com/108169988/217411594-f86adc33-4ff9-478e-8b73-e3e1be73e173.png)

Reply:
Hmm.... That is a really good suggestion! Let me try out the above!

Reply:
@sam-rice I changed the section to a nav element and wrapped the search input and nav links in a ul as li elements. Let me know what you think!

Reply:
Looks perfect to me!